packages/worker/src/replay.ts

```diff
@@ -81,7 +78,7 @@ export function handleReplayEviction(evictJob: RemoveFromCache, workflowId: stri
     // LANG_REQUESTED is used internally by Core to support duplicate runIds during replay.
     case EvictionReason.LANG_REQUESTED:
     case EvictionReason.CACHE_FULL:
```

Comment:
I'm surprised that we consider `cache_full` as a non-error. Can the replay worker legitimately ever send a `cache_full` event?